scripts/processFactory.ts

```diff
@@ -0,0 +1,525 @@
+﻿/// <reference path="../src/compiler/sys.ts" />
+module ts {
+    interface Map<T> {
+        [name: string]: T;
+    }
+
+    interface SyntaxNode {
+        name?: string;
+        kind?: string;
+        type?: string;
+        types?: string;
+        baseType?: string;
+        children?: SyntaxMember[];
+    }
+
+    interface SyntaxMember {
+        paramName?: string;
+        name?: string;
```

Comment:
I use it for Factory.createNumericLiteral so that it can generate the following:

```TypeScript
        export function createNumericLiteral(value: number | string, location?: TextRange, flags?: NodeFlags): LiteralExpression {
            var node = beginNode<LiteralExpression>(SyntaxKind.NumericLiteral);
            node.text = String(value);
            return finishNode(node, location, flags);
        }
```
Where `paramName` is `"value"` and `name` is `"text"`